packages/web-runtime/src/components/Topbar/Notifications.vue

```diff
@@ -175,21 +176,47 @@ export default {
         const space = store.getters['runtime/spaces/spaces'].find(
           (s) => s.fileId === messageRichParameters?.space?.id.split('!')[0] && !s.disabled
         )
-        if (space) {
+
+        if (space && subject === 'Removed from Space') {
```

Comment:
Isn't the subject translated by the backend? So this would only work for users who have set English as language?

Reply:
also... don't we have a different property to check on? Subject is really ugly for this :-(

Reply:
Haha yes, totally forgot 🙈 Well, then this is blocked by the server. There is no other way to distinguish between the subjects.

I'll create an issue for it.

Reply:
thank you 💪